readme.md

```diff
@@ -23,9 +23,19 @@ This project expects to be built using continuous integration in Azure Pipelines
 
```

Comment:
this filename is usually uppercase as README.md, rather than readme.md

Reply:
I agree. That came from the Defra boilerplate.